src/main/java/org/concordion/api/AbstractCommand.java

```diff
@@ -25,4 +25,9 @@ public boolean isExample() {
 
     public void executeAsExample(CommandCall commandCall, Evaluator evaluator, ResultRecorder resultRecorder) {
     }
+
+    public boolean alwaysRunEvenIfNoNonExampleChildren() {
+        return false;
+    }
+

```

Comment:
Oops. I mean "I'll change it to Execute and add JavaDoc to the Command.java interface where it's declared".